PersistentEmpiresLib/PersistentEmpiresLib/PersistentEmpiresMission/MissionBehaviors/PersistentEmpireBehavior.cs

```diff
@@ -43,6 +43,15 @@ public class MasterServerResponse
         public string ServerSignature;
 
         // public static string ServerSignature = "";
+        private static string _defaultClass = "pe_peasant";
+        public static string DefaultClass { get { return _defaultClass; } }
+
+        public static void SetDefaultClass(string defaultClass)
```

Comment:
Is this function actually called from anywhere? If not it can be read from generalconfig.xml

Reply:
Yes the idea is that other mods, based on PE, can use this function to set their own default class. I do it in PERP module, where I define my own class in own xml and want it to be used by PE as default :)